pkg/middleware/cache.go

```diff
@@ -56,7 +58,7 @@ func ServeCachedPage(ch *services.CacheClient) echo.MiddlewareFunc {
 
 			if err != nil {
 				switch {
-				case err == redis.Nil:
+				case errors.Is(err, &lib_store.NotFound{}) || err == redis.Nil:
```

Comment:
Can the check for `redis.Nil` be removed now?

Reply:
Probably but I wasn't 100% sure at the time so I left the nil check. Looking at the [gocache redis source](https://github.com/eko/gocache/blob/96fc2f790c4cfb27659863657625fa8c4e68d25d/store/redis/redis.go#L48-L50), it seems like it can be removed because `err == redis.Nil` is used as the check.